docs/public/static/diffs/template-bare-minimum/versions.json

```
@@ -0,0 +1 @@
+["43","44","45","46","47","48","49","unversioned"]
```

```suggestion
["46","47","48","49","unversioned"]
```

Let's remove legacy SDK and list only instructions for the listed in the Reference section versions.

---

@brentvatne and I had discussed this a while back and settled on 6 previous versions (though clearly 7 have shown up, there's an off-by-one bug in the generator!). My thinking was that it's easier to get behind more versions when managing your own native projects, and our upgrade instructions have longer shelf lives than our API reference.